README.md

```diff
@@ -49,7 +49,12 @@ faster.
 When stdin is nil or an empty string, the stdin device will get closed immediately.
 
 The `timeout` parameter speifies the timeout threshold for stderr/stdout reading timeout,
-the stdin writing timeout, and the process waiting timeout, respectively.
+the stdin writing timeout, and the process waiting timeout, respectively. If the timeout
+threshold is exceeded, the sub-process will be killed by first a SIGTERM. Then we will
```

Comment:
"by first a SIGTERM" is not proper English. I think you can write this:

> will be killed like this:
> 
> 1. first send a SIGTERM signal, and then
> 2. after 1 millisecond, sending a SIGKILL signal.

BTW, I do believe `1ms` is too soon. Many processes may take much longer than 1ms to clean up. How about 100ms instead?